### DISCUSSION TITLE
topology_scan_enabled config parameter

### DISCUSSION THREAD

**Discussion by pipiche38** [zigpy/zigpy#1012] [Q&A] [answered]
By default this parameter is set to True. What is the purpose when zigpy is not used in ZHA ?
What is the impact if we disable it ?

**Comment by dmulcahey** [comment, marked as answer]
https://github.com/zigpy/zigpy/blob/dev/zigpy/topology.py

it controls whether or not the topology info is periodically scanned and stored. This is used to do visualizations.

**Reply by pipiche38** [reply]
Thanks David for the response, so I beleive that if we don't want the visualization, we can simply disable ?
just wonder if there is internal code using also the neighbour tables (outside of visualisation)

**Reply by dmulcahey** [reply]
You should be able to disable it. I don’t know off the top of my head about your question but it should be very trivial / quick to look at the code to determine that. It’s all here…

**Reply by pipiche38** [reply]
yep, thanks for the quick reply.
